x-pack/plugin/analytics/src/main/java/org/elasticsearch/xpack/analytics/topmetrics/TopMetricsAggregator.java

```diff
@@ -234,12 +309,13 @@ public void swap(long lhs, long rhs) {
         @Override
         public Loader loader(LeafReaderContext ctx) throws IOException {
             // TODO allow configuration of value mode
```

Comment:
Should we put an info blurb somewhere in the docs that multi-valued fields will be averaged together?

Reply:
Probably! I'd planned to give the docs another pass in the next PR.